deps/build.jl

```diff
@@ -108,6 +108,34 @@ function find_ld_lld()
     return ""
 end
 
+function find_device_libs()
+    # The canonical location
+    if isdir("/opt/rocm/amdgcn/bitcode")
```

Comment:
Should we not do this check after we checked the environment variables? That would allow the user to override the location